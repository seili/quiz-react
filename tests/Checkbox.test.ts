// Link.react.test.js
import React from 'react';
import { CheckBox } from '../src/components/CheckBox/Checkbox';
import renderer from 'react-test-renderer'; 

test('Link changes the class when hovered', () => {
  const component = renderer.create(
    <CheckBox id="1" handleCheckChieldElement="" isChecked="" value=""/> 
  );
  let tree = component.toJSON();
});