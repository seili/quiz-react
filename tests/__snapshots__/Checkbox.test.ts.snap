// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<li>
  <input
    checked=""
    onClick=""
    type="checkbox"
    value=""
  />
   
  
</li>
`;
